packages/api/endpoints/granules.js

```diff
@@ -1229,6 +1379,7 @@ router.delete('/:collectionId/:granuleId', del);
 
 module.exports = {
   bulkDelete,
+  bulkMoveCollection,
```

Comment:
This is pretty minor/unimportant but would `bulkChangeCollection` or `bulkUpdateCollection` make more sense? I think "Move" is probably the most accurate but if I'm reading this endpoint for the first time and I don't know what the endpoint does, I feel like "MoveCollection" is less self-explanatory... I think.

Reply:
UI is important though, and naming things is hard. 

I don't hate bulk change collection...... 🤔